src/samples/plugins/interfaces/ISessionKeyPlugin.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import {UserOperation} from "@eth-infinitism/account-abstraction/interfaces/UserOperation.sol";
+
+interface IModularSessionKeyPlugin {
+    enum FunctionId {
+        RUNTIME_VALIDATION_TEMPORARY_OWNER,
+        USER_OP_VALIDATION_TEMPORARY_OWNER
+    }
+
+    /// @notice This event is emitted when a session key is added to the account.
+    /// @param account The account whose session key is updated.
+    /// @param sessionKey The address of the session key.
+    /// @param selector The selector of the function that the session key is allowed to call.
+    /// @param _after The time after which the owner is valid.
+    /// @param _until The time until which the owner is valid.
+    event SessionKeyAdded(
+        address indexed account, address indexed sessionKey, bytes4 selector, uint48 _after, uint48 _until
+    );
+
+    /// @notice This event is emitted when a session key is removed from the account.
+    /// @param account The account whose session key is updated.
+    /// @param sessionKey The address of the session key.
+    /// @param selector The selector of the function that the session key is allowed to call.
+    event SessionKeyRemoved(address indexed account, address indexed sessionKey, bytes4 selector);
+
+    /// @notice This event is emitted when session keys are added to the account.
+    /// @param account The account whose session keys are updated.
+    /// @param sessionKeys The addresses of the session keys.
+    /// @param selectors The selectors of the functions that the session keys are allowed to call.
+    /// @param _afters The times after which the owners are valid.
+    /// @param _untils The times until which the owners are valid.
+    event SessionKeysAdded(
+        address indexed account,
+        address[] sessionKeys,
+        bytes4[] selectors,
+        uint48[] _afters,
+        uint48[] _untils
```

Comment:
```suggestion
    /// @param afters The times after which the owners are valid.
    /// @param untils The times until which the owners are valid.
    event SessionKeysAdded(
        address indexed account,
        address[] sessionKeys,
        bytes4[] selectors,
        uint48[] afters,
        uint48[] untils
```

Reply:
Making `_after` to `after` leads to the following error

`Expected ',' but got reserved keyword 'after'`

So I changed all of it to `validAfter`, and same with all `until`s.